### ISSUE TITLE
[stable/keycloak] Feature request: Update Keycloak to 4.8.2.Final 

### ISSUE THREAD

**Issue by tsmgeek** [helm/charts#9450]
Keycloak has released 4.6.0-Final

Currently when I try to force the chart to use latest via `keycloak.image.tag` it does not work.
They have upgraded WildFly and this may be causing an issue when rolling upgrade tries to update keycloak-1 while keycloak-0 is still running old version.

https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-4-6-0-final

**Comment by tsmgeek** [comment]
Note that now 4.7.0 has been released.

**Comment by ysaakpr** [comment]
4,8.1 is out

**Comment by unguiculus** [comment]
I'll be looking into updating the chart soon.

**Comment by pushrbx** [comment]
FYI: I've been experimenting with this. I tried to use the latest image tag (atm 4.8.2) and it keeps crashing with 
```
VFS000002: Failed to clean existing content for temp file provider of type temp. Enable DEBUG level log to find what caused this
```

**Comment by tsmgeek** [comment]
4.8.2 is now out

**Comment by unguiculus** [comment]
Fixed in #10713
